packages/emotion/src/withStyle.tsx

```diff
@@ -101,15 +101,15 @@ export type WithStyleProps = Partial<{
  *
  * @module withStyle
  *
- * @param {function} generateStyle - The function that returns the component's style object
- * @param {function} generateComponentTheme - The function that returns the component's theme variables object
- * @returns {ReactElement} The decorated WithStyle Component
```

Comment:
Removing the types here removes them from the doc page too. https://613078228ea68c6766e640f1--preview-instui.netlify.app/#withStyle